cassandra/cluster.py

```diff
@@ -5012,12 +5012,16 @@ def exception_from_response(response):
                 return response.to_exception()
             else:
                 return response
+        if len(retry_decision) == 2:
+            retry_type, consistency = retry_decision
+            delay = None
+        elif len(retry_decision) == 3:
+            retry_type, consistency, delay = retry_decision
 
-        retry_type, consistency = retry_decision
         if retry_type in (RetryPolicy.RETRY, RetryPolicy.RETRY_NEXT_HOST):
             self._query_retries += 1
             reuse = retry_type == RetryPolicy.RETRY
-            self._retry(reuse, consistency, host)
+            self._retry(reuse, consistency, host, delay)
```

Comment:
Won't this result in undefined variable error if we take `if len(retry_decision) == 2:` branch?

Reply:
isn't `delay = None` enough as definition ?